docs/designs/link-output.md

```diff
@@ -57,20 +60,26 @@ Example:
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "/dotnet/articles/b"
+            "target_url": "/dotnet/articles/b",
+            "source_git_url": "https://github.com/dotnet/articles/a.md",
+            "source_line": "10"
         },
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "https://github.com"
+            "target_url": "https://github.com",
+            "source_git_url": "https://github.com/dotnet/articles/a.md",
+            "source_line": "20"
         },
         {
             "source_url": "/dotnet/articles/a",
             "source_moniker_group": "218c13d0",
-            "target_url": "./c.md"
+            "target_url": "./c.md",
+            "source_git_url": "https://github.com/dotnet/articles/a.md",
+            "source_line": "30"
         }
     ]
 }
 ```
-> Notice: `source_url` should be the root file for inclusion (TOC inclusion and markdown inclusion)
+> Notice: `source_url` should be the root file for inclusion (TOC inclusion and markdown inclusion), and `source_file_url` should be the included file
```

Comment:
would this cause too much work to v2 @superyyrrzz

Reply:
It is OK for v2.

The issue for v2 is it is hard to report line number under current implementation. If this is resolved, it is easier to report the line number from included file than from the including file.